### PR TITLE
status code update, post to put update, two routes for lastname

### PR DESCRIPTION
**Pull request by MarvellousUbani**
- changed status code from 201 to 200, 
- changed post for updates to put 
- added two routes for getting and setting last name of specific user

### CODE REVIEW

File: src/routes/index.js [hunk 1]
```diff
@@ -14,15 +14,20 @@ router.post('/users', catchErrors(userController.createUser));
...
 
 // More Routes
 //Get First Name
 router.get('/users/firstname/:id', catchErrors(userController.getUserFirstName));
 //Set First Name
-router.post('/users/firstname/:id', catchErrors(userController.setUserFirstname));
+router.put('/users/firstname/:id', catchErrors(userController.setUserFirstname));
+
+//Get Last Name
+router.get('/users/lastname/:id', catchErrors(userController.getUserLastName));
```
Comment (goodhands) [inline src/routes/index.js:28]: Please correct these routes to REST standard, it should be  <resource>/<id>/<property>:

``router.get('/users/:id/lastname', ...);``

Comment (MarvellousUbani) [reply src/routes/index.js:28]: Hey samuel,
Good catch.. Give me a sec to update this and push

Comment (MarvellousUbani) [reply src/routes/index.js]: And I fixed that...

**Comment by papa28x4** [comment]
Nice one, @MarvellousUbani  but please leave POST status code as 201. This indicates the creation of a resource was successful

**Pull request merged**